vermouth/pdb/pdb.py

```diff
@@ -99,12 +137,39 @@ def keyfunc(graph, node_idx):
 
 
 def write_pdb(system, path, conect=True, omit_charges=True):
+    """
+    Writes `system` to `path` as a PDB formatted string.
```

Comment:
There should be a word about the conect record being built from the edges. Latter on, it would be useful to add the ability to write the conect record from some interactions.